nodejs/readme.md

```diff
@@ -1,302 +1,115 @@
+[![LinkedIn](https://img.shields.io/badge/Follow_LinkedIn--white?logo=linkedin&style=social)](https://www.linkedin.com/company/tdengine)
+[![StackOverflow](https://img.shields.io/badge/Ask_StackOverflow--white?logo=stackoverflow&style=social&logoColor=orange)](https://stackoverflow.com/questions/tagged/tdengine)
 
-### DSN
+English | [简体中文](README-CN.md)
 
-User can connect to the TDengine by passing DSN to WebSocket client. The description about the DSN like before.
+## Table of Contents
+<!-- omit in toc -->
 
-```text
-[+<protocol>]://[[<username>:<password>@]<host>:<port>][/<database>][?<p1>=<v1>[&<p2>=<v2>]]
-|------------|---|-----------|-----------|------|------|------------|-----------------------|
-|   protocol |   | username  | password  | host | port |  database  |  params               |
-```
-
-- **protocol**: Display using websocket protocol to establish connection. eg. `ws://localhost:6041`
-- **username/password**: Database's username and password.
-- **host/port**: Declare host and port. eg. `localhost:6041`
-- **database**: Optional, use to specify database name.
-- **params**: Other parameters. Like cloud Token.
+- [Table of Contents](#table-of-contents)
+- [1. Introduction](#1-introduction)
+- [2. Documentation](#2-documentation)
+- [3. Prerequisites](#3-prerequisites)
+- [4. Build](#4-build)
+- [5. Testing](#5-testing)
+  - [5.1 Test Execution](#51-test-execution)
+  - [5.2 Test Case Addition](#52-test-case-addition)
+  - [5.3 Performance Testing](#53-performance-testing)
+- [6. CI/CD](#6-cicd)
+- [7. Submitting Issues](#7-submitting-issues)
+- [8. Submitting PRs](#8-submitting-prs)
+- [9. References](#9-references)
+- [10. License](#10-license)
 
-A complete DSN string example：
+## 1. Introduction
 
-```text
-ws://localhost:6041/test
-```
+@tdengine/websocket is an efficient connector specially designed by TDengine for Node.js developers. It uses the WebSocket API provided by the taosAdapter component to establish a connection with TDengine, eliminating the dependence on TDengine client drivers and opening up a convenient development path for developers. With this powerful tool, developers can easily build applications for TDengine clusters. Whether it is performing complex SQL write and query tasks, implementing flexible schemaless write operations, or achieving highly real-time subscription functionality, this connector can easily and perfectly meet diverse data interaction needs in all aspects.
 
-## Connection types
+## 2. Documentation
 
-**Node.js websocket connector** which is implemented through taosAdapter.
+- To use Node.js connector, please check [Developer Guide](https://docs.tdengine.com/developer-guide/), which includes how an application can introduce the `@tdengine/websocket`, as well as examples of data writing, querying, schemaless writing, parameter binding, and data subscription.
```

Comment:
Maybe better:

To use the Node.js connector, please refer to the [Developer Guide](https://docs.tdengine.com/developer-guide/), which includes instructions on how to integrate @tdengine/websocket into an application, along with examples of data writing, querying, schemaless writing, parameter binding, and data subscription.